bin/compile

```diff
@@ -63,7 +66,7 @@ fi
 
 # pip can't handle vcs & editable dependencies when requiring hashes (https://github.com/pypa/pip/issues/4995)
 # poetry exports local dependencies as editable by default (https://github.com/python-poetry/poetry/issues/897)
-poetry export -f "$REQUIREMENTS_FILE" "${EXPORT_PARAMETERS[@]}" > "$REQUIREMENTS_FILE.orig"
+poetry export -f requirements.txt "${EXPORT_PARAMETERS[@]}" -o "$REQUIREMENTS_FILE.orig" | indent
```

Comment:
Why not keep `"$REQUIREMENTS_FILE"` here?

Reply:
Right, why this change? Then you don't need to extend poetry mock. This is the reason why I didn't use `-o` even though I knew about it.

Reply:
It made it look like you could put any path there, when in reality the only option is the exact string `requirements.txt` as an output format. https://python-poetry.org/docs/cli/#options-12

Reply:
Oh, that's actually a good point. Thank you! And how about the `-o` part?

Reply:
The issue I had with not using `-o` was that it put a “Creating virtualenv…” string at the top of every the output file because which came through stdout: https://github.com/python-poetry/poetry/issues/6275

So if you just pipe the output, pip fails later when it tries to install the first line of requirements.txt.

Reply:
@davegaeddert it looks like it's just a convenience variable in case the file name changes from `requirements.txt` in the future. If it changes, it only changes in one line of the code.

It's set [on line 54](https://github.com/moneymeets/python-poetry-buildpack/pull/44/files/34a423d6ae9074e42f817829ba87853040970d46#diff-a64a7d9b82578ca3053416d56631a61e0ceabed87619415663030e3842429e05R54)

Reply:
I'm not sure about the merits of `-o` vs `>`, so I'll default to @zyv on that. I'm curious though.

Reply:
I see, so they have added output that isn't going to stderr like before, but mixes with stdout. Extending the mock is annoying, but it seems that's indeed the best way. Thanks!